tests/CMakeLists.txt

```diff
@@ -5,36 +5,42 @@
 # at once with CMake's ctest command line tool from the build folder.
 enable_testing()
 
+################################################################################
+# General tests and tools
```

Comment:
We can consider moving this CMakeLists.txt file to the unit tests directory and moving aviftest.c and avifyuv.c back to the top-level CMakeLists.txt file.

I seem to remember one reason we added this CMakeLists.txt file was to allow us to compile the C++ test files with different compiler flags.